src/ui/qgsmapsavedialog.ui

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>QgsMapSaveDialog</class>
+ <widget class="QDialog" name="QgsMapSaveDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>600</width>
+    <height>225</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Save map as image</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <layout class="QGridLayout" name="gridLayout">
+     <item row="3" column="0" colspan="2">>
+      <widget class="QCheckBox" name="mDrawAnnotations">
+       <property name="text">
+        <string>Draw annotation(s)</string>
+       </property>
+       <property name="checked">
+        <bool>true</bool>
+       </property>
+      </widget>
+     </item>
+     <item row="2" column="0" colspan="2">
+      <widget class="QCheckBox" name="mDrawDecorations">
+       <property name="text">
+        <string>Draw active decoration(s)</string>
```

Comment:
Any chance to provide a list of active decorations (in brackets or the tooltip)?